sdk/src/connections/usb/linux/usb_depth_sensor_linux.cpp

```diff
@@ -467,6 +479,18 @@ aditof::Status UsbDepthSensor::writeAfeRegisters(const uint16_t *address,
     using namespace aditof;
     Status status = Status::OK;
 
+    if (address == nullptr) {
+        LOG(ERROR) << "Received AfeRegisters address null pointer";
+        return Status::INVALID_ARGUMENT;
+    }
+
+    if (data == nullptr) {
+        LOG(ERROR) << "Received AfeRegisters data null pointer";
+        return Status::INVALID_ARGUMENT;
+    }
+
+    assert(length > 0);
```

Comment:
`length` has the type `size_t` which by definition is unsigned, so this is not needed.

Reply:
In this way we will prevent the length to be set to 0.